R/ForecastRegrTask.R

```diff
@@ -0,0 +1,58 @@
+#' @title Create a task for univariate forecasting
+#'
+#' @rdname Task
+#'
+#' @description Creates a task for univariate forecasting learners
+#'
+#' @export
+makeForecastRegrTask = function(id = deparse(substitute(data)), data, target,
+  weights = NULL, blocking = NULL, frequency = 1L, fixup.data = "warn",
+  check.data = TRUE) {
+  assertString(id)
+  assertClass(data, "xts")
```

Comment:
As I understand it, `ForecastRegrTask` is the *only* task that takes data *not* in the form of a `data.frame`? It would seem more consistent to me if it also took a `data.frame`. If that is impractical, take care to change the documentation of the `data` argument in `Task.R`.